scripts/mytop.sh

```diff
@@ -1099,15 +1182,15 @@ sub GetData()
 
         $lines_left--;
 
-        my($data) = Hashes('show global variables like "read_only"');
+        my($data) = Hashes('SHOW /*mytop*/ GLOBAL VARIABLES LIKE "read_only"');
         if ($data->{Value} ne "OFF")
         {
             print RED() if ($HAS_COLOR) ;
             print " ReadOnly";
 	    RESET() if ($HAS_COLOR);
         }
 
-	($data) = Hashes('SHOW SLAVE STATUS');
+	($data) = Hashes('SHOW /*mytop*/ SLAVE STATUS');
```

Comment:
There are also other parts of code where indentation is violated, L:1198, 1211,...